Tests/test_Align_codonalign.py

```diff
@@ -8231,13 +8231,12 @@ def test_dn_ds(self):
         self.assertAlmostEqual(dS, 0.0164, places=4)
 
         try:
-            import scipy
+            from scipy.linalg import expm
```

Comment:
If scipy is present, but expm cannot be imported, then that is a test failure, and should not be skipped.